### PR TITLE
Release docker image for arm64

### LINKED ISSUE #31: ARM64 Support
Hi Team,

I am trying to use “helm” over ARM64v8 but it seems it does not have an arm64 supported tag in docker hub. After the below changes, the image was successfully uploaded to Dockerhub.

-  **Dockerfile:** Added a new build argument ARCH which will install the binaries as per the architecture.

-  **.travis.yml:** Added architecture arm64-graviton2 and installed jq in .travis.yml file to build, test, and release image for arm64.

-  **Build.sh:** Added check so that the image for arm64 will be pushed with name <image name>:<tag>-arm64 and also provided the architecture as build argument for building the image for a particular architecture.

**Commit Link:** https://github.com/Fayeal/helm/commit/86a79a6b989ceecdc357c7275996412d48c2481f

**Travis Link**: https://travis-ci.com/github/Fayeal/helm/builds/213286679

**Dockerhub Link:** https://hub.docker.com/repository/docker/odidev/helm

It looks that you are using travis-ci.org and as per https://blog.travis-ci.com/2020-09-11-arm-on-aws the arm64-graviton support is added for travis-ci.com and travis is also shutting down travis-ci.org.

It will be very helpful if this image is released for the arm64 platform. If interested, I can raise a PR. But first, we need to migrate this project from travis-ci.org to travis-ci.com.

Please let me know if any help is required.

### PR DESCRIPTION
**Pull request by odidev**
Resolves #31.

The following files have been modified:

- Added a new build argument TARGETARCH in the **Dockerfile** which will decide which architecture binary to install.

- In **.travis.yml** added “dist:focal” in order to use buildx and installed buildx binary and created a new builder and switched to it.

- I have used buildx to build and push the image for both platforms in the **build.sh** file.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -11,27 +11,9 @@
...
-    echo "matched"
-  else
-    echo "unmatched"
-    exit
-  fi
-
   if [[ "$TRAVIS_BRANCH" == "master" ]]; then
     docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-    docker push ${image}:${tag}
+    docker buildx build --platform linux/amd64,linux/arm64 --build-arg VERSION=${tag} -t ${image}:${tag} --push .
```
Comment (ozbillwang) [inline build.sh:16]: Do we need a `--push` at build stage?

second, with `docker buildx build` , we need another command to build and generate image locally for testing. The image could be suitable your local OS, so you can make sure the image works..

**Comment by barthy1** [comment]
Hi @ozbillwang 
any progress on this PR? 
I would be happy to introduce other archs support once this initial update is merged :)

**Comment by ozbillwang** [comment]
No updates after I left the comments.

**Comment by odidev** [comment]
@ozbillwang,

Please review and let me know if any changes are required.

**Comment by ozbillwang** [comment]
@odidev 

I did the review, but doesn't see your update.

image

**Comment by odidev** [comment]
@ozbillwang 

Sorry for the confusion, but I am unable to view your review comments anywhere in the PR or commit.

As per your [comment](https://github.com/alpine-docker/helm/pull/33#issuecomment-780982859), it looks that you want to run **docker build** on both platforms and then test the image. If you want to build the image for both the platforms using docker build then run the test and after that push to Dockerhub. For this, I'll suggest you please refer to my changes mentioned in the [comment](https://github.com/alpine-docker/helm/issues/31#issue-786864206).

**Commit Link:** Fayeal@86a79a6

If changes look good, I can amend the above changes in this PR.

Further, as mentioned in your comment for redesigning the code to support multi-arch via **docker buildx build** command, and in order to implement buildx I have removed the testing part from the code, I think testing the image with buildx is not feasible.

**Comment by ozbillwang** [comment]
@odidev 

Are you ok that I take over the change, and raise a new PR for both of you to review?

**Comment by odidev** [comment]
@ozbillwang 

Yes, I am happy to review.

**Comment by ozbillwang** [comment]
Thanks, @odidev 

I am thinking to write a generic script/tool to build multi-arch iamges, so I don't have to update the `build.sh` for each of the alpine images under this Org. 

It would take time for the first version, please wait.

In the mean time, build your own images, so I wouldn't block you.

**Comment by ozbillwang** [comment]
closed by https://github.com/alpine-docker/helm/pull/37

**Pull request closed without merging**